package-configuration/linux-cli/get-cloudify.py

```diff
@@ -340,6 +344,22 @@ def install_pycrypto(self, venv):
         run(cmd)
 
 
+def check_cloudify_installed(virtualenv_path=None):
+    if virtualenv_path:
+        result = run('{0}{1}{2}'.format(
+            virtualenv_path, ENV_BIN_RELATIVE_PATH,
```

Comment:
os.join.path or this will break if user provides path with forward slash in the end `/path/to/venv`